internal/groups/api/responses.go

```diff
@@ -0,0 +1,151 @@
+package groups
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/mainflux/mainflux"
+)
+
+var (
+	_ mainflux.Response = (*memberPageRes)(nil)
+	_ mainflux.Response = (*groupRes)(nil)
+	_ mainflux.Response = (*groupDeleteRes)(nil)
+	_ mainflux.Response = (*assignMemberToGroupRes)(nil)
+	_ mainflux.Response = (*removeMemberFromGroupRes)(nil)
+)
+
+type pageRes struct {
+	Total  uint64 `json:"total"`
+	Offset uint64 `json:"offset"`
+	Limit  uint64 `json:"limit"`
+	Name   string `json:"name"`
+}
+type memberPageRes struct {
+	pageRes
+	Members []interface{}
+}
+
+func (res memberPageRes) Code() int {
+	return http.StatusOK
+}
+
+func (res memberPageRes) Headers() map[string]string {
+	return map[string]string{}
+}
+
+func (res memberPageRes) Empty() bool {
+	return false
+}
+
+type viewGroupRes struct {
+	ID          string                 `json:"id,omitempty"`
+	Name        string                 `json:"name,omitempty"`
+	ParentID    string                 `json:"parent_id,omitempty"`
+	OwnerID     string                 `json:"owner_id,omitempty"`
+	Description string                 `json:"description,omitempty"`
+	Metadata    map[string]interface{} `json:"metadata,omitempty"`
+	// Level if retrieved with Parents or Children method
+	// indicates a level in hierarchy from first group node.
+	Level    int             `json:"level,omitempty"`
+	Path     string          `json:"path"`
```

Comment:
A short explanation similar to the one for `level` field would be nice. Also, is level empty (thus, omitted) when if it's not Parents/Children method?

Reply:
path and level are always there